packages/reactive-element/src/reactive-element.ts

```diff
@@ -1083,7 +1092,7 @@ export abstract class ReactiveElement
    * @return Returns a node into which to render.
    * @category rendering
    */
-  protected createRenderRoot(): Element | ShadowRoot {
+  protected createRenderRoot(): HTMLElement | DocumentFragment {
```

Comment:
Why change `ShadowRoot` to `DocumentFragment`, as `ShadowRoot` seems to be the more specific return from `this.attachShadow`?

Reply:
Just because lit-html's render can render to a DocumentFragment. It seems best to align all these types.